### DISCUSSION TITLE
Extracting Match Winner from CloudBet API After It's Resulted

### DISCUSSION THREAD

**Discussion by malookshah4** [cloudbet/docs#75] [General]
Hello everyone,

How can I determine the winner of a match using the CloudBet API after the match has ended, and which API endpoint should I use for this purpose?

I already try this (https://sports-api.cloudbet.com/pub/v2/odds/events/20689565) endpoint with match_id but there is nothing about match winner.  check out below example.

{
  "sequence": "0",
  "id": 20690080,
  "sport": {
    "name": "Tennis",
    "key": "tennis"
  },
  "competition": {
    "name": "ITF Tunisia 46A, Women Singles",
    "key": "tennis-itf-women-itf-tunisia-46a-women-singles",
    "category": {
      "name": "ITF Women",
      "key": "itf-women"
    }
  },
  "home": {
    "name": "Anja Wildgruber",
    "key": "c7cb41-anja-wildgruber",
    "abbreviation": "WIL",
    "nationality": "DEU",
    "researchId": "31c3df6b-dbe1-42ac-9c92-153537b62188"
  },
  "away": {
    "name": "Aliona Falei",
    "key": "c7327a-falei-aliona",
    "abbreviation": "FAL",
    "nationality": "BLR",
    "researchId": "95161d22-1ac2-4af5-b07f-e3f2c64bf003"
  },
  "status": "RESULTED",
  "markets": {},
  "name": "Anja Wildgruber V Aliona Falei",
  "key": "c7cb41-anja-wildgruber-v-c7327a-falei-aliona",
  "cutoffTime": "2023-11-02T11:30:00Z",
  "metadata": {
    "opinions": {
      "tennis.winner": {
        "outcomes": [
          {
            "outcome": "away",
            "probability": 0.39602113,
            "marketKey": "tennis.winner",
            "params": ""
          },
          {
            "outcome": "home",
            "probability": 0.60397893,
            "marketKey": "tennis.winner",
            "params": ""
          }
        ]
      }
    },
    "opinion": [
      {
        "outcome": "away",
        "probability": 0.39602113,
        "marketKey": "tennis.winner",
        "params": ""
      },
      {
        "outcome": "home",
        "probability": 0.60397893,
        "marketKey": "tennis.winner",
        "params": ""
      }
    ]
  },
  "type": "EVENT_TYPE_EVENT",
  "settlement": {},
  "resultedTime": "2023-11-02T13:35:20Z",
  "gradingDuration": 327,
  "endTime": "2023-11-02T13:08:27Z"
}

**Comment by kgravenreuth** [comment]
Hi @malookshah4 ,

Apologies for the delay in reply. 

As mentioned in [this discussion](https://github.com/Cloudbet/docs/discussions/19), we can provide you information about how your API bets have resulted via the [Bet History](https://www.cloudbet.com/api/?utm_source=github.com&utm_medium=community&utm_campaign=axis&urls.primaryName=Trading#/Trading/BetHistory) endpoint of our Trading API. 

Note that you need to be registered as a Cloudbet player to use the Trading API key with a minimum of 10 euro deposit as mentioned in our [Trading API blog article](https://www.cloudbet.com/en/blog/posts/sports-betting-api-tutorial-with-golang#toc-0?utm_source=github.com&utm_medium=community&utm_campaign=axis)

For example, If you'd like to know the result of a given event, then you can place bets on the Winner or Moneyline markets for that event and then check the bet results to know the event's results.

Regards,
Klaus Gravenreuth
Product & Engineering @ Cloudbet

**Reply by RomainTess** [reply]
Hello. Late to this discussion but still have a question. 
Does this mean that the only way to get the winner of a specific event after it has ended is by checking the bet result?

**Reply by kgravenreuth** [reply]
Hi @RomainTess ,
Yes, at the moment you would have to place bets and then check the bet results to know the event's results.

Regards,
Klaus Gravenreuth
Product & Engineering @ Cloudbet
